agent/src/com/jonnyzzz/teamcity/virtual/run/docker/DockerContext.java

```diff
@@ -16,13 +16,18 @@
 
 package com.jonnyzzz.teamcity.virtual.run.docker;
 
+import com.intellij.openapi.util.SystemInfo;
```

Comment:
It is more reliable to use `jetbrains.buildServer.agent.BuildAgentConfiguration#getSystemInfo` to get agent OS instead. Currently you are using semi-deprecated API